### ISSUE TITLE
[Accessibility] "Comment as.." buttons should not have tabindex = -1

### ISSUE THREAD

**Issue by DaazKu** [vanilla/vanilla#4829]
The buttons are not accessible using the keyboard only.
We should make sure this is not the case and add the proper tabindex key if the normal flow if weird.

**Comment by linc** [comment]
I can find no logical reason it's like this. It appears with tabindex -1 on the first creation of the function with no explanation. 30643132993684e64761f457eae2ab304e960ec7

**Comment by linc** [comment]
> Be warned, however, that tabIndex can only safely be used on certain elements and the tabIndex should typically be set to zero (0). When a tabIndex greater than zero (0) is used, the implementation must be essentially perfect for every active element of the page for it to succeed, and is therefore very error-prone. 

https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Attribute/tabindex

Looks like I should just delete it
